csv_to_openfisca.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+import pandas as pds
+import numpy as np
+
+# from openfisca_core.simulations import Simulation
+from openfisca_core.simulation_builder import SimulationBuilder, Simulation
+from openfisca_country_template import CountryTaxBenefitSystem
+
+tax_benefit_system = CountryTaxBenefitSystem()
+
+def set_members_entity_id(entity, entity_id_array):
```

Comment:
Such a method should probably be put somewhere in Core, as it seems to be general enough to justify abstracting the numpy magics behind it.